### DISCUSSION TITLE
Replacing the AdBlocker

### DISCUSSION THREAD

**Discussion by peternrdstrm** [bromite/bromite#1553] [Ideas]
As most of us know, by now, Bromite's adblocker is mediocre at best. The DAT format's size limit and not supporting CSS rules are two of the biggest weaknesess, I personally see.  Both privacy and user-experiencewise, it bottlenecks the user, having to deal with cookie consent dialogs, popups and rarely slip-through ads. I'm sure some improvement could (and maybe should) be made to the default filter list, as a long term idea, it might be better, to entirely replace the ad-blocking engine. Sure, the addition of userscripts helps, but for me, popup/cookie consent blockers weren't working propperly/at all. 
Now an argument can be made, that a more advanced ad-blocker might slow Bromite down, but realistically, Bromite is by far the fastest android browser, I have ever tried (both on a flagship and on a low-end device). Also, on paper a more-advanced adblocker would improve page load times, and might even save more battery and bandwidth. 
Also, if the same GUI-less style is kept, I don't think general performance would take a noticable hit. 
I hope you may consider this as an option, as Bromite has been the absolute best browser I've ever used, and I really hope it can/will get even better.

**Comment by OmlineEditor** [comment]
to solve the problem, you need to allow other blockers to be installed in the form of plugins. there are no plugins yet :-(
